pom.xml

```
@@ -355,7 +355,7 @@
 						</goals>
 						<configuration>
 							<inputSpec>
-								https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
+								https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-2827-mybank-transactionId/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
```

```suggestion
								https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
```
TODO to be replaced after dependent PR merge